rfcs/003-randomness.md

```diff
@@ -0,0 +1,210 @@
+# RFC-003: Randomness
+
+Author: José Duarte — @jmg-duarte
+Date: 6/12/24
+
+## Abstract
+
+This document covers existing randomness methods in Polkadot and discusses which
```

Comment:
Love the abstract <3 It gives the quick summary, specifically tells what this is about and what decision we're making.